client/verta/verta/_cli/registry/update.py

```diff
@@ -106,6 +106,14 @@ def update_model_version(model_name, version_name, label, model, custom_module,
 
 
 def add_attributes(model_version, attribute, overwrite):
-    _multiple_arguments_for_each(attribute, "attribute",
-                                 lambda key, value: model_version.add_attribute(key, json.loads(value), overwrite=overwrite),
+    def action(key, value):
+        try:
+            model_version.add_attribute(key, json.loads(value), overwrite=overwrite)
+        except ValueError as e:
+            if not (value.startswith('\"')  and value.endswith('\"')):
+                raise click.BadParameter("if the attribute's value is a string, consider wrapping it in quotes.")
+            else:
+                raise e
```

Comment:
The code in a `try` block should be as minimal as possible (so that the `except` actually catches what we want it to catch). We're trying to catch an error loading JSON, so the `try` block should only consist of loading JSON.
```suggestion
        try:
            json_value = json.loads(value)
        except ValueError as e:
            if not (value.startswith('\"')  and value.endswith('\"')):
                raise click.BadParameter("if the attribute's value is a string, consider wrapping it in quotes.")
            else:
                raise e
        else:
            model_version.add_attribute(key, json_value, overwrite=overwrite)
```

Reply:
Fixed.